store/etcd/types_transformation.go

```diff
@@ -0,0 +1,182 @@
+package etcd
+
+import (
+	"fmt"
+
+	"github.com/heetch/regula"
+	"github.com/heetch/regula/rule"
+	pb "github.com/heetch/regula/store/etcd/proto"
+)
+
+// toProtobufRuleset creates a protobuf Ruleset from a regula.Ruleset.
+func toProtobufRuleset(rs *regula.Ruleset) *pb.Ruleset {
```

Comment:
:gb: In english you'd the subject first, then the rest: `rulesetToProtobuf`, `rulesetFromProtobuf`, etc.

Reply:
Done in https://github.com/heetch/regula/pull/86/commits/a188bdf15426995c35f699969b09d42b9581b95d.